### PR TITLE
cap internal bleeding wounds at 1 per organ

### PR DESCRIPTION
**Pull request by d3athrow**
suck my diiiiick

### CODE REVIEW

File: code/modules/organs/organ_external.dm [hunk 1]
```diff
@@ -256,9 +256,15 @@
...
 	var/local_damage = brute_dam + burn_dam + damage
 	if(damage > 10 && type != BURN && local_damage > 20 && prob(damage) && is_organic() && !(owner.species && owner.species.flags & NO_BLOOD))
-		var/datum/wound/internal_bleeding/I = new (15)
-		wounds += I
-		owner.custom_pain("You feel something rip in your [display_name]!", 1)
+		var/internal_bleeding = 0
+		for(var/datum/wound/Wound in wounds)
+			if(Wound.internal)
+				internal_bleeding = 1
+			break
```
Comment (N3X15) [inline code/modules/organs/organ_external.dm]: break needs to be under the if statement.  +1 indent.

**Comment by N3X15** [comment]
:+1:

**Comment by NigglyWiggly** [comment]
But what about all the fun of watching all of your blood drain out of you over a minute or two?

**Comment by Probe1** [comment]
its not a significant issue to me.  merge and observe how it goes

**Comment by SangriaMonkey** [comment]
Good stuff

**Comment by Shadowmech88** [comment]
It's not exactly realistic, but I still think it could be a good change.
External bleeding could use some changes as well, later.

**Comment by Rei1226** [comment]
god bless

**Comment by clusterfack** [comment]
Rude branch

**Comment by clusterfack** [comment]
Just wondering, since wound is just an abstraction. How 'big' is an internal organ wound ingame? How fast does it bleed?

**Pull request merged**